app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -134,6 +134,17 @@ def download_statistics
     end
   end
 
+  # Gets the 10 most recent past answers of a specific question
+  def past_answers
```

Comment:
You might need to do some authorization on this as well.

Reply:
Actually this can be put under a new controller - `Course::Assessment::SubmissionQuestion::SubmissionQuestionController` - since it deals with getting the past answers from a particular submission question. Easier to handle the authorization and necessary details. 

Then you will need to add this method in SubmissionQuestion model:
```ruby
def answers
  Course::Assessment::Answer.where('submission_id = ? AND question_id = ?',
                                     self.submission_id, self.question_id)
end
```